### PR TITLE
Add Sequence.next/1 for quickly creating sequences

### LINKED ISSUE #66: Make a sequence/1 function that generates a default string
A lot of times I just want a unique value like this:

```
title: sequence(:title, fn(n) -> "Title #{n}" end)
body: sequence(:comment_body, fn(n) -> "Comment Body #{n}" end)
```

For simple cases like this I think it would be helpful if you could do something like

```
title: sequence(:title) #generates "Title #{n}"
body: sequence(:comment_body) #generates "Comment Body #{n}"
```

Thoughts on this? Would it be better to use Faker or would this be useful?

### PR DESCRIPTION
**Pull request by paulcsmith**
Closes #66

### CODE REVIEW

File: lib/ex_machina/sequence.ex [hunk 1]
```diff
@@ -3,6 +3,12 @@ defmodule ExMachina.Sequence do
     Agent.start_link(fn -> HashDict.new end, name: __MODULE__)
   end
 
+  def next(sequence_name) do
```
Comment (jsteiner) [inline lib/ex_machina/sequence.ex]: Should we raise a helpful error for non-string values?

**Comment by jsteiner** [comment]
One question/comment. Otherwise :+1:

File: lib/ex_machina/sequence.ex (hunk 1 above)
Comment (paulcsmith) [inline lib/ex_machina/sequence.ex]: That's a good point. I'm not sure though. Should we raise, or should we convert it to a string? I think raise, but just want to make sure :)

Comment (jsteiner) [inline lib/ex_machina/sequence.ex]: I think we should raise. I think anything else would be unexpected and unlikely to be helpful (if not harmful).

Comment (paulcsmith) [inline lib/ex_machina/sequence.ex]: I think you're right. I'll make it raise :)

**Pull request merged**